src/main/java/com/mraof/minestuck/entity/item/GristEntity.java

```diff
@@ -59,6 +69,18 @@ public GristEntity(EntityType<? extends GristEntity> type, Level level)
 		super(type, level);
 	}
 	
+	/**
+	 * this is where we set up our consumedelay
+	 */
+	public GristEntity(Level level, double x, double y, double z, GristAmount gristData, int pickupDelay)
+	{
+		this(level, x, y, z, gristData);
+		consumeDelay = pickupDelay;
+		// Set the class's consume-delay variable to equal the pickupDelay value that got passed in.
```

Comment:
Comments describing an action should go above the action being described.

Reply:
honestly i think i'm gonna re comment all of our code since my commenting etiquite was off back when this was made

Reply:
Go for it if you want, but this comment makes sense to me!

Reply:
oh yeah this was after i got it. those big blocks of text that just explain my thoughts on the feature are just annoying tho